<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>matrix</class>
 <widget class="QWidget" name="matrix">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>350</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>6</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Kernel</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/img/icon/icon.png</normaloff>:/img/icon/icon.png</iconset>
  </property>
  <widget class="QPushButton" name="pb00">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb01">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb06">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb05">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb02">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>0</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb03">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>0</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb04">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>0</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb15">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb11">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb14">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb12">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb13">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb10">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb16">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb25">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb21">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb24">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb22">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb23">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb20">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb26">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb35">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb31">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb34">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb32">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb33">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb30">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb36">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>150</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb45">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb41">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb44">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb42">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb43">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb40">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb46">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>200</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb55">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb51">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb54">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb52">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb53">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>250</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb50">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb56">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb65">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>300</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb61">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb64">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>300</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb62">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb63">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>300</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb60">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>300</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb66">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>300</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
